src/components/v2/Table/styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+import { AnyStyledComponent } from 'styled-components';
+
+export const useStyles = () => {
+  const theme = useTheme();
+  return {
+    tableContainer: css`
+      background-color: transparent;
+      background-image: none;
+      box-shadow: none;
+    `,
+    table: css`
+      min-width: 650px;
```

Comment:
I don't think this is related to the storybook decorator more than the minimum width on the table. I would think we would want a minimum width on the table to keep things spaced nicely and usable with a horizontal scroll. 

Or we need a mobile layout to display on smaller screens

Reply:
The thing is that this minimum width isn't based on the actual width of the content itself, since it's set as a fixed value. If we display a table that has only two columns for example, it would still be 650px wide which will make it huge.

Being able to apply a minimum width to the table is a nice idea I think, I would just make it a bit more modular. For example, we can make the table take the full available horizontal width, and add a `css` prop to the component which will apply to the container, so that a component/page rendering it can easily extend its style and so set a minimum width if needed.